.github/container/Dockerfile.base

```diff
@@ -38,12 +38,24 @@ RUN <<"EOF" bash -ex
 git config --global user.name "${GIT_USER_NAME}"
 git config --global user.email "${GIT_USER_EMAIL}"
 EOF
-RUN pip install --upgrade --no-cache-dir pip pip-tools && rm -rf ~/.cache/*
 RUN mkdir -p /opt/pip-tools.d
 ADD --chmod=777 \
     get-source.sh \
     pip-finalize.sh \
     /usr/local/bin/
+RUN wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_$(dpkg --print-architecture) -O /usr/local/bin/yq && \
```

Comment:
[question] Will it be better to install it thru `apt install`:
```
$> add-apt-repository ppa:rmescandon/yq
$> apt update
$> apt install yq -y
```
?

Reply:
I guess it would be better to have it be managed by `apt`, then the package can be inspected by `dpkg`. FWIW, I had to install `apt-get install software-properties-common` to get `add-apt-respository` which is 68.4MB according to `apt`. Is that okay with you?